### DISCUSSION TITLE
Restart lab

### DISCUSSION THREAD

**Discussion by taraska90** [srl-labs/containerlab#1692] [Q&A] [answered]
Hello, dear community.
Lab: [Two SR Linux nodes](https://containerlab.dev/lab-examples/two-srls/)
I am sorry in advance for maybe a silly question, but what is the right way to start the lab after the host machine is rebooted? I see only deploy -t with the reconfigure option,  but even with deploy save, after rebooting the host VM and running deploy --reconfigure srlinux config are cleared and new containers for srl created. There is another option to start srl containers?

**Comment by hellt** [comment, marked as answer]
Hi
if you just want to start the lab and nodes to start with the config as you saved it before destroying, then doing just `containerlab deploy -t <topo>` should work.

There has been reported an issue #1685 that claims that the lab directory is always created, despite old lab directory being present.
We will have a look at it, could be that this is a regression that needs to be fixed.

**Reply by taraska90** [reply]
Hi, thank you. It's strange, but the configuration was saved with `commit now save` even after redeploy.
Anyway, I will wait for a newer release.  

p.s. It is not clear from the documentation what is the right way to stop the lab. I can help to write the article if needed.

**Reply by hellt** [reply]
yes, it has been identified that the issue is only present when `startup-config` is used, which is the case for the lab you're trying to run.

We will work on the fix in the next release.

To stop the lab you do `containerlab destroy`, this will remove the containers, but keep the last saved state (like configs saved with `commit save` or the likes).

If you do `containerlab destroy --cleanup` both the containers and lab state will be gone.
It is briefly explained here in the [quickstart](https://containerlab.dev/quickstart/#destroying-a-lab), but I agree it makes sense to add more color to it from information in this [doc page](https://containerlab.dev/manual/conf-artifacts/).
